changelogs/unreleased/-1vju5j1.yml

```diff
@@ -0,0 +1,6 @@
+---
+title: Added Side Navigation with route navigating to blank components (#1vju5j1)
+ticket_id: "#1vju5j1"
+merge_request: 16
+author: meet-aniket
```

Comment:
Update this.
Update your git config

Reply:
should i update my username with my full name in git config

Reply:
Updated sir